ruby_event_store/lib/ruby_event_store/client.rb

```diff
@@ -189,9 +190,13 @@ def with_metadata(metadata, &block)
       self.metadata = previous_metadata
     end
 
+    def load_serialized_event(event_type:, event_id:, data:, metadata:)
+      mapper.load_serialized_event(event_type: event_type, event_id: event_id, data: data, metadata: metadata)
```

Comment:
Because I think it's faster to go directly from serialized attributes to event. There is nothing special in SerializedRecord here that we need, it would be extra allocation, this is how ActiveRecord become useless... extra allocations everywhere.